integration_tests/run_pyspark_from_build.sh

```diff
@@ -174,7 +174,16 @@ else
     else
         TEST_PARALLEL_OPTS=("-n" "$TEST_PARALLEL")
     fi
-    RUN_DIR=${RUN_DIR-"$SCRIPTPATH"/target/run_dir}
+
+    # Delete old run directories
+    cd "$SCRIPTPATH/target"
+    for f in */; do
+      if [[ $f = run_dir* && $(($(date +%s) - ${f:8:(${#f} - 9)})) -gt 86400 ]]; then
+        rm -rf "$f"
+      fi
+    done
+
+    RUN_DIR=${RUN_DIR-"$SCRIPTPATH"/target/run_dir-$(date +%s)}
     mkdir -p "$RUN_DIR"
```

Comment:
We can do it with mktemp in a single line and probably easier for the user to read the calendar date instead of epoch. In addition it has a random suffix to avoid collisions which is good for CI 
```suggestion
    mktemp -p ${RUN_DIR-"$SCRIPTPATH"/target} -d run_dir-$(date +%Y%m%d%H%M%S)-XXXX
```

Reply:
Thank you! I will update this.